artemis-image/pom.xml

```diff
@@ -0,0 +1,127 @@
+
+
+   <dependencies>
+      <dependency>
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>apache-artemis</artifactId>
+         <version>${project.version}</version>
+         <type>pom</type>
+      </dependency>
+      <dependency>
+         <groupId>junit</groupId>
+         <artifactId>junit</artifactId>
+         <version>${junit.version}</version>
+         <scope>test</scope>
+      </dependency>
+      <!-- The johnzon-core and json-api contents are repackaged in -commons,
+           However maven can still need them during tests that don't see the shaded bits during build -->
+      <dependency>
+         <groupId>org.apache.johnzon</groupId>
+         <artifactId>johnzon-core</artifactId>
+         <scope>test</scope>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-api</artifactId>
+         <scope>test</scope>
+      </dependency>
+   </dependencies>
+
+   <build>
+      <plugins>
+         <plugin>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>jib-maven-plugin</artifactId>
+            <version>3.3.0</version>
+            <configuration>
+               <from>
+                  <!-- reproducible build and single pull with @sha reference -->
+                  <image>eclipse-temurin:19-jre@sha256:619e1c33be065f75eb00b0999627b7b1e070f55c427e63dfd256f06d00935111</image>
+               </from>
+               <to>
+                  <image>target/activemq-artemis-image:${project.version}</image>
+               </to>
+               <container>
+                  <mainClass>org.apache.activemq.artemis.ActiveMQImageMain</mainClass>
+                  <ports>
+                     <port>61616</port>
+                  </ports>
+                  <format>OCI</format>
+               </container>
+            </configuration>
+            <executions>
+               <execution>
+                  <phase>package</phase>
+                  <goals>
+                     <goal>buildTar</goal>
+                  </goals>
+               </execution>
+            </executions>
+         </plugin>
+
+         <!-- to easily create a single salted credential for our ${example.user.file}-->
+         <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>${exec-maven-plugin.version}</version>
+            <configuration>
+               <executable>java</executable>
+               <arguments>
+                  <argument>-classpath</argument>
+                  <classpath/>
+                  <argument>org.apache.activemq.artemis.spi.core.security.jaas.SCRAMPropertiesLoginModule</argument>
+                  <argument>${example.user}</argument>
+                  <argument>${example.pwd}</argument>
+               </arguments>
+               <outputFile>${example.user.file}</outputFile>
+            </configuration>
+         </plugin>
```

Comment:
similarly seems example-specific, feels off having it in the main pom of a deployed module.